kvdb-rocksdb/src/iter.rs

```diff
@@ -78,22 +84,35 @@ pub trait IterationHandler {
 
 	/// Create an `Iterator` over the default DB column or over a `ColumnFamily` if a column number
 	/// is passed.
-	fn iter(&self, col: u32) -> Self::Iterator;
+	/// In addition to a read lock and a column index, it takes a ref to the same `ReadOptions` we
```

Comment:
Not sure why it talks about the read lock here? That is not part of the interface?

Same below.